src/components/Global/Analytics/TopPools.tsx

```diff
@@ -0,0 +1,226 @@
+                                ? 'text-positive'
+                                : 'text-negative'
+                        }`}
+                    >
+                        {poolApy ? poolApy + '%' : '...'}
+                    </div>
+                </td>
+                <td
+                    className=' px-6  whitespace-nowrap'
+                    style={{ height: '40px' }}
+                >
+                    <div className=' text-text1'>{poolVolume ?? '...'}</div>
+                </td>
+                <td
+                    className='hidden lg:table-cell px-6  whitespace-nowrap'
+                    style={{ height: '40px' }}
+                >
+                    <div
+                        className={`text-text1 ${
+                            isPoolPriceChangePositive
+                                ? 'text-positive'
+                                : 'text-negative'
+                        }`}
+                    >
+                        {poolPriceChangePercent ?? '...'}
+                    </div>
+                </td>
+                <td
+                    className='px-6 whitespace-nowrap'
+                    style={{ height: '40px' }}
+                >
+                    <div className='flex items-center justify-end h-full'>
+                        <button
+                            className='flex items-center justify-center text-bg-dark1 border-dark3 border rounded-full hover:text-accent1 hover:border-accent1'
+                            style={{
+                                width: '48px',
+                                height: '25px',
+                            }}
+                        >
+                            <span className='px-2 py-1'>Trade</span>
+                        </button>
+                    </div>
+                </td>
+            </tr>
+        );
+    };
+
+    return (
+        <div className='flex flex-col h-full'>
+            <div className='flex-grow overflow-auto h-full hide-scrollbar'>
+                <div className='py-2 h-full'>
+                    <div className='shadow rounded-lg bg-dark1 h-full py-2'>
+                        <table className='divide-y divide-dark3 relative w-full '>
+                            <thead
+                                className=' sticky top-0 h-25 '
+                                style={{ height: '25px', zIndex: '2' }}
+                            >
+                                <tr className='text-text2 text-body font-regular capitalize leading-body'>
+                                    <th
+                                        scope='col'
+                                        className='sticky top-0 px-8 text-left tracking-wider'
+                                    >
+                                        Tokens
+                                    </th>
+                                    <th
+                                        scope='col'
```

Comment:
These guys can probably be one component reused

Reply:
Some of those tds are sm:table-cell, others xl:table-cell. It all depends on the importance of  that column. As the screen gets smaller, we remove some of them so while the code might look the same, there is a difference in a few of them.